flake.nix

```diff
@@ -1,140 +1,33 @@
-# The flake.nix is the entrypoint of all nix code.
-#
-# This repository uses the standard tool https://github.com/divnix/std.
-# Familiarity with std is required to be able to contribute effectively.
-# While official documentation for std can be found in its GitHub, this flake
-# has been thoroughly commented so as to quickstart new maintainers.
-# This flake can also be used as a template for new std-based projects.
-# Further documentation can be found in nix/README.md
-#
-# You may want to refer to the standard glossary as you go along:
-# https://divnix.github.io/std/glossary.html
 {
   description = "Plutus Core";
 
+
   inputs = {
-    nixpkgs = {
-      follows = "haskell-nix/nixpkgs";
-    };
-    std = {
-      url = "github:divnix/std";
-      inputs.nixpkgs.follows = "nixpkgs";
-    };
-    haskell-nix = {
```

Comment:
I suspect I will want to pin `haskell.nix` also, but I guess this is fine for now

Reply:
Will do